tutorials/performance/using_servers.rst

```diff
@@ -119,6 +119,32 @@ This is an example of how to create a sprite from code and move it using the low
         var xform = Transform2D().rotated(deg_to_rad(45)).translated(Vector2(20, 30))
         RenderingServer.canvas_item_set_transform(ci_rid, xform)
 
+ .. code-tab:: csharp
+
+    public partial class MyNode2D : Node2D
+    {
+        // RenderingServer expects references to be kept around. GC will otherwise remove your objects sooner or later...
```

Comment:
Not sure if the mention of the GC is needed.
```suggestion
        // RenderingServer expects references to be kept around.
```

Reply:
because the GC doesnt immediately remove the object, beginners might tend to NOT keep the reference because it works even without the reference. but this might only be a few seconds and they will run into problems later. i find it an extremly helpful remark/further insight.